README.md

```diff
@@ -56,7 +107,25 @@ klepto steal \
 --read-max-conns=8
 ```
 
-## Relationships
+Behind the scenes Klepto will establishes the connection with the source and target databases with the given parameters passed, and will dump 4 the tables.
+
+We recommend to always set the following parameters:
+- `concurrency` to alleviate the pressure over both the source and target databases.
+- `read-max-conns` to limit the number of open connections, so that the source database does not get overloaded.
+
+<a name="configuration-file-options"></a>
+## Configuration File Options
+
+You can set the following fields in the file:
+* Tables
+* Matchers
+* more?
```

Comment:
What other fields can you set in the toml file, and what do they mean?

Reply:
- `Tables` represents table definition
- `Filters` represents the way you want to filter the results
- `Matchers` are variables to replace placeholders in Filter
- `Relationships` represents a relationship definition